src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/LockFileSnippets.cs

```diff
@@ -28,6 +28,56 @@ internal static class LockFileSnippets
             }}";
         }
 
+        public static string CreateCrossTargetingLockFileSnippet(
+            string[] targets,
+            string[] originalTargetFrameworks,
+            string[] targetFrameworks,
+            string[] libraries = null,
+            string[] projectFileDependencyGroups = null,
+            string[] logs = null)
+        {
+            return $@"{{
+  ""version"": 3,
+  ""targets"": {{{string.Join(",", targets)}}},
+  ""libraries"": {{{(libraries == null ? string.Empty : string.Join(",", libraries))}}},
+  {GetLogsPart(logs)}
+  ""projectFileDependencyGroups"": {{{(projectFileDependencyGroups == null ? string.Empty : string.Join(",", projectFileDependencyGroups))}}},
+  ""project"": {{
+    ""version"": ""1.0.0"",
+    ""restore"": {{
+      'projectUniqueName': 'C:\\git\\repro\\consoletest\\consoletest.csproj',
+      'projectName': 'consoletest',
+      'projectPath': 'C:\\git\\repro\\consoletest\\consoletest.csproj',
+      ""packagesPath"": ""C:\\Users\\saoslund\\.nuget\\packages\\"",
```

Comment:
You should probably remove this. I suspect it might trip up the credential scanner